news/2024/2024-01-28-vocaloid-mapping-contest-2.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Vocaloid Mapping Contest 2 & New Featured Artist: iroha(sasaki)"
+date: 2024-01-28 12:00:00 +0000
+---
+
+VOCALOID lovers rejoice! The Vocaloid Mapping Contest returns bigger and better than ever, featuring support for every game mode, an improved format, a bit of necromancy, and incredible music from our brand-new Featured Artist: **iroha(sasaki)**!
+
+![](/wiki/Contests/VMC/2/img/banner.jpg)
+
+Yep, you heard right. **iroha(sasaki)**, the artist behind the legendary VOCALOID classic *Meltdown* featured in *many* classic osu! maps, is now a Featured Artist (and the 400th one, no less)!
+
+*Meltdown* has already earned its place in osu!'s history with several popular Ranked maps, including [Diasios](https://osu.ppy.sh/users/25999)'s [early map of *Meltdown*](https://osu.ppy.sh/beatmapsets/9070#osu/36814) from 2009, [ertyukjh010](https://osu.ppy.sh/users/220939)'s [map and unique storyboard](https://osu.ppy.sh/beatmapsets/45672#osu/142590) from 2012, and even more recently [ohm002](https://osu.ppy.sh/users/4468239)'s [modern marathon](https://osu.ppy.sh/beatmapsets/1743736#osu/3564761) from 2022!
+
+Regardless of how popular *Meltdown* may be, it's far from **iroha(sasaki)**'s only song — we're happy to announce that **11** songs are part of [**iroha(sasaki)**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/399)!
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/399/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+***BUT WAIT!!!*** This is more than just a Featured Artist announcement!
+
+## Vocaloid Mapping Contest 2
+
+The [**Vocaloid Mapping Contest** (***VMC***)](/wiki/Contests/VMC) returns, and just like last time, it's all about spotlighting hidden VOCALOID gems!
```

Comment:
include the contest listing link somewhere here for ease of access

Reply:
put in the "big changes" section